pybliometrics/scopus/serial_title.py

```diff
@@ -129,7 +141,7 @@ def title(self):
         """The title of the source."""
         return self._entry['dc:title']
 
-    def __init__(self, issn, refresh=False, view="ENHANCED"):
+    def __init__(self, issn, refresh=True, view="ENHANCED", date=None):
```

Comment:
Can we rename the new parameter to `years` please? I think that's more intuitive. Also, have it after the issn parameter please. This underlines the parameter's importance

Reply:
I'm also not so happy that here refresh should be True by default. It makes sense, yes, but it's violates the consistency with all the other classes.

Reply:
> Can we rename the new parameter to `years` please? I think that's more intuitive. Also, have it after the issn parameter please. This underlines the parameter's importance

sure no problem

Reply:
> I'm also not so happy that here refresh should be True by default. It makes sense, yes, but it's violates the consistency with all the other classes.

I'll bring it back to false and force refresh only when years is specified